onnx/defs/tensor/defs.cc

```diff
@@ -170,11 +170,11 @@ OPERATOR_SCHEMA(Gather)
     .NumOutputs(1)
     .SetDoc(R"DOC(
 Given `data` tensor of rank r >= 1, and `indices` tensor of rank q, gather
-entries of the outer-most dimension of `data` indexed by `indices`, and concatenate
+entries of the axis dimension of `data` (usually outer-most one as axis=0) indexed by `indices`, and concatenates
```

Comment:
nit: maybe change "usually" to "default"?

Another general question I have is whether we should mention axis can be negative?